### PR TITLE
Feat: Add home page

### PR DESCRIPTION
**Pull request by starver20**
Added landing page with images containing links to respective categories.
[CSB](https://ex5kz.csb.app/homePage/homePage.html) for reference

### CODE REVIEW

**Comment by gadgetfather** [comment]
Everything looks good! A small suggestion would be, you can replace that watermarked image
image

**Review by gadgetfather — APPROVED** [review]

File: homePage/homePage.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  grid-template-columns: 1fr 1fr;
+  gap: var(--spacing-20);
+}
+
+.category-shoe,
+.category-jersey {
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+
```
Comment (VishalPatil18) [inline homePage/homePage.css]: Empty lines could be avoided.

**Review by VishalPatil18 — APPROVED** [review]
- I might be wrong here but in my opinion, it's not a good idea to put up a motivational quote on the landing page of an E-commerce site. What you could do is add this motivational quote with some product that would do the branding of the product and add that as a carousel on the landing page. I think that would make more sense. Although it's just a suggestion, the final call relies on you.

- The navbar can be made fixed/sticky as that would help in better UX for the site.

> The landing page looks great! Good stuff @starver20 👍

**Comment by VishalPatil18** [comment]
Also, one thing to ask here is, why did the Netlify checks fail?

**Comment by starver20** [comment]
> Also, one thing to ask here is, why did the Netlify checks fail?

It failed because there is no index.html file in the root of my project, netlify always looks for a index.html file first.

**Comment by starver20** [comment]
> Everything looks good! A small suggestion would be, you can replace that watermarked image image

Yes, I will do that

**Comment by starver20** [comment]
> 

I was thinking of a carousel, but i left it for later bcoz it would need some js.
I will make the navbar sticky, thanks for the suggestions.

**Pull request merged**